### PR TITLE
update nxos readme

### PR DESCRIPTION
**Pull request by samccann**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

documentation
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
+
+# Cisco NXOS Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.nxos) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/cisco.nxos)-->
+
+The Ansible Cisco NXOS collection includes a variety of Ansible content to help automate the management of Cisco NXOS network appliances.
+
+This collection has been tested against Cisco NXOS 7.3.(0)D1(1) on VIRL.
```
Comment (NilashishC) [inline README.md:8]:
```suggestion
This collection has been tested against Cisco NX-OS 7.0(3)I5(1).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    ansible-galaxy collection install cisco.nxos
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: cisco.nxos
+    version: 0.0.1
```
Comment (NilashishC) [inline README.md:29]:
```suggestion
    version: 1.0.0
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+
+
+# Cisco NXOS Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.nxos) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/cisco.nxos)-->
+
+The Ansible Cisco NXOS collection includes a variety of Ansible content to help automate the management of Cisco NXOS network appliances.
+
```
Comment (cidrblock) [inline README.md:7]: Can we add

# Content Overview

The goal of the NXOS connection plugin when combined with NXOS resource modules (link here) is to provide an experience for network operators and engineers a familiar experience for NXOS that aligns with the other core networking platform supported by Ansible. Ongoing development efforts and contributions to this collection are solely focused on enhancements to current resource modules, additional resource modules and enhancements to connection plugins.

Comment (samccann) [reply README.md:7]: I added the first apart here. The ongoing development part I added to the contributing section.

**Pull request merged**